mgmt/rest/client/client_func_test.go

```diff
@@ -100,9 +100,10 @@ func startAPI() (string, error) {
 	}
 	s.SetMetricManager(c)
 	s.Start()
-	r.BindConfigManager(c.Config)
 	r.BindMetricManager(controlClient)
 	r.BindTaskManager(client)
+	configClient, _ := control.NewConfigClient(c.Config.ListenAddr, c.Config.ListenPort)
```

Comment:
We should catch the error here and kill running the tests if we can't create a client for some reason.